backend/src/test/java/de/bund/digitalservice/ris/caselaw/integration/tests/AuthIntegrationTest.java

```diff
@@ -56,9 +56,6 @@ void testGetUserNameWithoutKnownGroup() {
         .uri("/api/v1/auth/me")
         .exchange()
         .expectStatus()
-        .isOk()
-        .expectBody(User.class)
-        .consumeWith(
-            response -> assertThat(response.getResponseBody().name()).isEqualTo("testUser"));
```

Comment:
hat sich in diesem Rahmen an der Implementierung irgendwas geändert, dass hier auf einmal ein 4xx zurückkommt?